shell/platform/windows/win32_window.cc

```diff
@@ -131,12 +132,28 @@ Win32Window::MessageHandler(HWND hwnd,
         break;
 
       case WM_MOUSEMOVE:
+        if (!tracking_mouse_events) {
+          TRACKMOUSEEVENT tme;
+          tme.cbSize = sizeof(tme);
+          tme.hwndTrack = hwnd;
+          // Not tracking Hover since the engine handles that logic.
```

Comment:
I was confused by this comment at first, not already being familiar with the APIs here. I think it would be clearer to a non-Windows-expert (which is probably most of the people dealing with this code, in practice) to remove this comment and instead comment the whole block as "Register for an event when the mouse leaves the window". Or even better, extract it to a helper function so it's self-documenting and out of the main flow.

Reply:
Done